code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -547,11 +548,11 @@
 	switch(skin)
 		if("ointment")
 			new /obj/item/weapon/storage/firstaid/fire/empty(Tsec)
-		if("tox")
+		if(TOX)
 			new /obj/item/weapon/storage/firstaid/toxin/empty(Tsec)
 		if("o2")
 			new /obj/item/weapon/storage/firstaid/o2/empty(Tsec)
-		if("brute")
+		if(BRUTE)
```

Comment:
I don't think you meant to change these.

Reply:
The define BRUTE is the same as "brute" so it makes sense to use the existing defines.